otter/test/utils.py

```diff
@@ -370,6 +370,16 @@ def __init__(self, code, headers, data=None):
         # Data is not part of twisted response object
         self._data = data
 
+    def __eq__(self, other):
+        return (
+            isinstance(other, self.__class__) and
+            self.code == other.code and
+            self.headers == other.headers and
+            self._data == other._data)
+
+    def __neq__(self, other):
+        return self != other
```

Comment:
This doesn't actually work, so I'm guessing it's missing some test coverage.

1. The magic method for inequality is spelled `__ne__`, not `__neq__`
2. To implement inequality in terms of equality, you want to do `not (a == b)`, not with `!=`, otherwise you get a recursion error.
3.